pkg/master/master.go

```diff
@@ -156,6 +179,52 @@ type completedConfig struct {
 	*Config
 }
 
+func (c *Config) createMasterCountReconciler() EndpointReconciler {
+	// use a default endpoint reconciler if nothing is set
+	endpointClient := coreclient.NewForConfigOrDie(c.GenericConfig.LoopbackClientConfig)
+	return NewMasterCountEndpointReconciler(c.MasterCount, endpointClient)
+}
+
+func (c *Config) createLeaseReconciler() EndpointReconciler {
+	ttl := c.MasterEndpointReconcileTTL
+	config, err := c.StorageFactory.NewConfig(kapi.Resource("apiServerIPInfo"))
+	if err != nil {
+		glog.Fatalf("Error determining service IP ranges: %v", err)
```

Comment:
Is this an acceptable pattern to exit the application at this point instead of returning an error?

Reply:
Yes, I think so. It is during config setup